webapp/backend/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "Backend for NSMQ AI web application",
+  "main": "server.ts",
+  "scripts": {
+    "test": "npm run build && npm run jasmine",
+    "start": "ts-node-esm src/server.ts",
+    "tsc": "tsc",
+    "build": "npx tsc",
+    "prettier": "prettier --config .prettierrc \"src/**/**.ts\" --write",
+    "lint": "eslint \"src/**/**.ts\"",
+    "jasmine": "jasmine",
+    "watch": "tsc-watch --esModuleInterop src/server.ts --outDir ./build --onSuccess \"node ./build/server.js\""
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nsmq-ai/nsmqai.git"
+  },
+  "keywords": [
+    "nsmq"
+  ],
+  "author": "NSMQ Web app team",
+  "license": "ISC",
```

Comment:
is the license here correct? can you verify that it matched what we have here https://github.com/nsmq-ai/nsmqai/blob/main/LICENSE

Reply:
Modified license in package.json file.